apis/config/v1beta1/configuration_types.go

```diff
@@ -226,3 +230,24 @@ type Integrations struct {
 	//  - "kubeflow.org/tfjob"
 	Frameworks []string `json:"frameworks,omitempty"`
 }
+
+type QueueVisibility struct {
+	// ClusterQueues is configuration to expose the information
+	// about the top pending workloads in the cluster queue.
+	ClusterQueues *ClusterQueueVisibility `json:"clusterQueues,omitempty"`
+
+	// UpdateIntervalSeconds specifies the time interval for updates to the structure
+	// of the top pending workloads in the queues.
+	// The minimum value is 1.
+	// Defaults to 5s.
+	UpdateIntervalSeconds int32 `json:"updateIntervalSeconds,omitempty"`
```

Comment:
Can we make it `time.Duration`? As we did for `CacheSyncTimeout` or `WaitForPodsReady,Timeout`. It is more flexible.

Reply:
It was done before, but as I understand we should follow api-convension 
Please, see the conversation here: https://github.com/kubernetes-sigs/kueue/pull/1069#discussion_r1300173369

Reply:
Got it, keep it as is then and thanks for the reference!